robyn/__init__.py

```diff
@@ -48,6 +48,8 @@ def __init__(
         self.config = config
         self.dependencies = dependencies
 
+        dev_mode_env_var = os.getenv("ROBYN_DEV_MODE", False) == "True"
```

Comment:
`__file__, file_object, __name__`

Try using these variables if possible

Reply:
```
print(__file__)
print(__name__)
print(file_object)
```

```
(.venv) RobynApp $ python3 app.py
.venv/lib/python3.12/site-packages/robyn/__init__.py
robyn
app.py

(.venv) RobynApp $ robyn app.py
.venv/lib/python3.12/site-packages/robyn/__init__.py
robyn
app.py

(.venv) RobynApp $ robyn -m src
.venv/lib/python3.12/site-packages/robyn/__init__.py
robyn
src/__main__.py

(.venv) RobynApp $ python3 -m src
.venv/lib/python3.12/site-packages/robyn/__init__.py
robyn
src/__main__.py

```

nops, no help :(

Reply:
@VishnuSanal , I have made an update to solve it.